README.md

```diff
@@ -30,6 +34,8 @@ The primary goal is to provide an interface that WASI programs can use to read a
 
 Although I2C is in some aspects not that different from SPI, the purpose of this proposal is to solely focus on I2C.
 
+Currently, behaviour that is not specified by this API is seen as undefined and, unless there's sufficient interest, is regarded as out-of-scope.
```

Comment:
Could you expand on what kind of behavior this is meant to cover?

My thought is, we don't need to specify the full semantics of the I<sup>2</sup>C protocol in WASI; we just need to specify how the WASI APIs map to I<sup>2</sup>C operations. With that understanding, it seems like a lot of places where there's ambiguity in terms of what the exact behavior is can be left to the I<sup>2</sup>C standards to figure out.